polling/frequent/workflow.go

```diff
@@ -0,0 +1,30 @@
+package frequent
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+func FrequentPolling(ctx workflow.Context) (string, error) {
+	logger := workflow.GetLogger(ctx)
+
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout: 60 * time.Second,
+		HeartbeatTimeout:    2 * time.Second,
+	}
+	ctx = workflow.WithActivityOptions(ctx, ao)
+
+	// @@@SNIPSTART samples-go-polling-frequent-activity
```

Comment:
I don't think we need SNIP comments as these are not referenced from documentation.